server/tests/Parties.test.js

```diff
@@ -47,6 +47,144 @@ describe('GET API/V1/PARTIES/:PARTY-ID', () => {
     }
   });
 
+  it('should return failure status 400', (done) => {
+    try {
+      chai.request(app)
+        .post('/api/v1/parties')
+        .set('x-auth-token', generateJwtToken(1, process.env.ADMIN_EMAIL, 'True'))
+        .send({
+          name: 'N',
+          logoUrl: 'http://someurl',
+          hqAddress: 'a given address here.'
+        })
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.property('status');
+          const returnStatus = 400;
```

Comment:
You can define this globally since you will be using it repeatedly in each test suite